silq/pulses/pulse_sequences.py

```diff
@@ -1263,6 +1245,8 @@ class NMRCPMGPulseSequence(NMRPulseSequence):
     Notes:
         For given pulse settings, `NMRPulseSequence.generate` will recreate the
         pulse sequence from settings.
+
+
     """
```

Comment:
In the actual function definition, I would also make the default example a Hahn echo style experiment.

In the NMR settings I would something like:
```python
self.pulse_settings['NMR'] = self.NMR = {
            'stage_pulse': DCPulse('empty'),
            'NMR_pulse': SinePulse('NMR_pi_half'),
            'NMR_DD_pulse': SinePulse('NMR_pi').
            'NMR_pulses': ['NMR_pulse', 'NMR_DD_pulse', 'NMR_pulse'],
            'pre_delay': 5e-3,
            'inter_delay': 1e-3,
            'post_delay': 2e-3,
            'final_phase': 0,
            'artificial_frequency': 0
        }

```

Season to taste. :wink:

Reply:
yeap, makes sense!